.vscode/settings.json

```diff
@@ -7,5 +7,25 @@
         "out": true // set this to false to include "out" folder in search results
     },
     // Turn off tsc task auto detection since we have the necessary tasks as npm scripts
-    "typescript.tsc.autoDetect": "off"
+    "typescript.tsc.autoDetect": "off",
+    "workbench.colorCustomizations": {
```

Comment:
regarding these colors, I don't know how to exclude them partially from settings. I am using peacock to get different colors for different VCS instances. But it stores colors into settings.json and not into a dedicated file, which could be excluded via .gitignore
https://github.com/johnpapa/vscode-peacock/issues/7